examples/action_bodies/search_delete_by_subject.json

```diff
@@ -0,0 +1,5 @@
+{
+    "delete_by": "subject",
+    "search_index": "dea6af9a-cab8-4a2c-a8ba-5b7c73334d3b",
```

Comment:
Do you want a real (or real-ish) index id here, or a dummy one like the all 0's in the other example?

Reply:
Good suggestion!